include/revng/ABI/AArch64/AAPCS64.h

```diff
@@ -70,7 +71,15 @@ struct Trait<model::ABI::AAPCS64> {
 
   static constexpr bool CalleeIsResponsibleForStackCleanup = false;
   static constexpr size_t StackAlignment = 16;
-  static constexpr size_t MinimumStackArgumentSize = 8;
+  static constexpr std::array ScalarStackAlignment{
```

Comment:
I suspect a previous commit was using this.

Reply:
I'll look into it.

Reply:
This is irrelevant because of recent redesign. I'm closing the thread.